src/System.IO.FileSystem.DriveInfo/src/System/IO/DriveInfo.Unix.cs

```diff
@@ -62,8 +88,18 @@ public long AvailableFreeSpace
             [SecuritySafeCritical]
             get
             {
-                Interop.libc.statfs data = Interop.libc.GetStatFsForDriveName(Name);
-                return (long)data.f_bsize * (long)data.f_bavail;
+                Interop.Sys.MountPointInformation mpi;
+                int result = Interop.Sys.GetSpaceInfoForMountPoint(Name, out mpi);
+                if (result == 0)
+                    return mpi.AvailableFreeSpace;
+                else
+                {
+                    Interop.ErrorInfo err = Interop.Sys.ConvertErrorPlatformToPal(result).Info();
```

Comment:
Again use new ErrorInfo. I won't comment on all of them if there are any more. Search for ConverError in your change.